To Do List/js/app.js

```diff
@@ -0,0 +1,143 @@
+// Select the Elements
+const clear = document.querySelector(".clear");
+const dateElement = document.getElementById("date");
+const list = document.getElementById("list");
+const input = document.getElementById("input");
+
+// Classes names
+const CHECK = "fa-check-circle";
+const UNCHECK = "fa-circle-thin";
+const LINE_THROUGH = "lineThrough";
+
+// Variables
+let LIST, id;
+
+// get item from localstorage
+let data = localStorage.getItem("TODO");
+
+// check if data is not empty
+if(data){
+    LIST = JSON.parse(data);
+    id = LIST.length; // set the id to the last one in the list
+    loadList(LIST); // load the list to the user interface
+}else{
+    // if data isn't empty
+    LIST = [];
+    id = 0;
+}
+
+// load items to the user's interface
+function loadList(array){
+    array.forEach(function(item){
+        addToDo(item.name, item.id, item.done, item.trash);
+    });
+}
+
+// clear the local storage
+clear.addEventListener("click", function(){
+    localStorage.clear();
+    location.reload();
+});
+
+// Show todays date
+const options = {weekday : "long", month:"short", day:"numeric"};
+const today = new Date();
+
+dateElement.innerHTML = today.toLocaleDateString("en-US", options);
+
+// add to do function
+
+function addToDo(toDo, id, done, trash){
+    
+    if(trash){ return; }
+    
+    const DONE = done ? CHECK : UNCHECK;
+    const LINE = done ? LINE_THROUGH : "";
+    
+    const item = `<li class="item">
+                    <i class="fa ${DONE} co" job="complete" id="${id}"></i>
+                    <p class="text ${LINE}">${toDo}</p>
+                    <i class="fa fa-trash-o de" job="delete" id="${id}"></i>
+                  </li>
+                `;
+    
+    const position = "beforeend";
+    
+    list.insertAdjacentHTML(position, item);
+}
+
+// add an item to the list user the enter key
+document.addEventListener("keyup",function(even){
+    if(event.keyCode == 13){

```

Comment:
I would recommend extracting Key Codes to `const`'s e.g. `const ENTER_KEY = 13` makes the code easier to read.